www/src/pages/components/fab/android/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Fab
+description: Buttons providing easy access to important actions.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Usage
+
+FAB stands for floating action button and they perform a specific highly important task. FABS are usually high up in the page’s hierarchy and as such should be used sparingly to avoid overuse.
+
+### Components
+
+There are two types of FAB components: ThumbprintIconFab and ThumbprintTextFab
+
+ThumbprintIconFab sit in the lower right ahd side of a UI and perform important tasks that are high priority compared to other actions on the page.
+
+ThumbprintTextFab can have text and an optional icon left of the text.
```

Comment:
Add a note about placement for this too? Should be in bottom middle of the page according to https://docs.google.com/document/d/1yms-6TGSliYF0I3nyX5oxH-GOWORRgXyxJseVh0LJuM/edit

Reply:
Nice, I was using the figma and wasn't aware of this doc.